### PR TITLE
Pt/admin export orders

### PR DESCRIPTION
**Pull request by priscilatrevino**
An admin can view and export orders into a csv file that were placed within the time frame of when the admin created an office lunch order and the day the admin will export the orders. 
_Note: *The reason a user shows as having multiple orders rather than one order with multiple items is because we currently don't have the functionality of a user being able to create an order with multiple items._ 

Screen Shot 2022-08-08 at 12 46 36 PM
Screen Shot 2022-08-09 at 2 54 55 PM
Screen Shot 2022-08-08 at 12 47 52 PM

### CODE REVIEW

File: lib/lunchbot_web/controllers/admin/export_orders_controller.ex [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          timestamp,
+          user,
+          for item <- order_items do
+            item
+          end
+        ]
+        |> List.flatten()
+      end
+
+    IO.inspect(all_orders, label: "ALL ORDERS")
```
Comment (chewnoill) [inline lib/lunchbot_web/controllers/admin/export_orders_controller.ex]: :eyes:

File: lib/lunchbot_web/controllers/admin/export_orders_controller.ex [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    all_orders =
+      for order <- order_struct do
+        timestamp = order.inserted_at
+        user = order.user.name
+
+        order_items =
+          for order_item <- order.order_items do
+            item_with_options =
+              for order_item_option <- order_item.order_item_options do
```
Comment (chewnoill) [inline lib/lunchbot_web/controllers/admin/export_orders_controller.ex]: there are a lot of nested for loops here, maybe we can push some of this logic into a SQL query?

**Comment by cathyzoller** [comment]
hold off on merging... check for nil lunch order id is in progress.

**Review by cathyzoller — APPROVED** [review]
Great job Priscila!

**Pull request merged**